docs/index.ts

```diff
@@ -0,0 +1,97 @@
+const navMenuButton = document.getElementById("coffee-menu");
+const menuButton = document.getElementById("button-icon-burger");
+const navModal = document.getElementById("navModal")!;
+const sliderContainer = document.getElementById("sliderContainer")!;
```

Comment:
Is the exclamation necessary? We use it sometimes, but also should be avoided, as it increases the risk of mistakes